types/account_test.go

```diff
@@ -0,0 +1,28 @@
+package types
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestAccount(t *testing.T) {
+
+	acc := Account{
+		PubKey:   nil,
+		Sequence: 0,
+		Balance:  nil,
+	}
+
+	//test Copy
+	accCopy := acc.Copy()
+	accCopy.Sequence = 1
+	t.Log(acc.Sequence)
+	t.Log(accCopy.Sequence)
+	assert.True(t, acc.Sequence != accCopy.Sequence, "Account Copy Error")
```

Comment:
okay switched. was confused because the pointer wasn't equal but that makes sense, not supposed to be.